lib/media/track/mediatrack.js

```diff
@@ -159,6 +159,9 @@ class MediaTrack extends Track {
       this._attachments.add(el);
     }
 
+    // enable the track;
+    this.mediaStreamTrack.enabled = true;
```

Comment:
@makarandp0 What if the publisher has disabled the Track? Can we make sure that we take that into account?

Reply:
not sure what you mean - Shouldn't this be independent of what publisher does with the track.  This property is only controlling if the track is allowed to render or not. 

```
The enabled property on the MediaStreamTrack interface is a Boolean value which is true if the track is allowed to render the source stream or false if it is not. This can be used to intentionally mute a track. When enabled, a track's data is output from the source to the destination; otherwise, empty frames are output.
```